Click

```diff
@@ -0,0 +1,2 @@
+def click():
```

Comment:
Should this file have a filetype? It looks like python code, so I would guess that `Click` should instead by `Click.py`